README.md

```diff
@@ -294,14 +294,12 @@ Install PyTorch
 
 ```bash
 export CMAKE_PREFIX_PATH="${CONDA_PREFIX:-'$(dirname $(which conda))/../'}:${CMAKE_PREFIX_PATH}"
-python -m pip install -r requirements.txt
 python -m pip install --no-build-isolation -v -e .
```

Comment:
Oh no, when did we start recommending pip install -e . This will mess up users' ccache setup unless they use the CCACHE_NOHASH envvar.

Reply:
@zou3519 has been complaining to me that vllm's build always happens in an unpredictable directory. Perhaps this is specifically a problem with their build backend, so maybe it's fine for us.